scripts/release/utils.injector.js

```diff
@@ -0,0 +1,249 @@
+#!/usr/bin/env node
+import chalk from 'chalk';
+import json2md from 'json2md';
+import fs from 'node:fs';
+import path from 'node:path';
+
+import { Build } from '../../documents/scripts/paths.js';
+import { ELEMENT_DIST, ELEMENT_SOURCE, generateDocList, sanitize } from './util.js';
+
+/**
+ * Return full text comment from Signature Typedoc structure.
+ * @param signature type from typedoc
+ * @returns {string} text comment
+ */
+const getComment = (signature) => {
+  if (signature?.comment?.summary?.length < 0) return '';
+  return signature?.comment?.summary.map((item) => item.text).join('');
+};
+
+/**
+ * Return full Return text from Signature Typedoc structure.
+ * @param signature type from typedoc
```

Comment:
"type from typedoc" would mean `signature` is a type, not a `Signature` defined in typedoc.
```suggestion
 * @param signature typedoc's Signature
```